cmd/dbfactory/cluster.go

```diff
@@ -120,12 +120,13 @@ func newSearchCommand(svc rdsiface.RDSAPI) *cobra.Command {
 			}
 
 			instanceType := ""
+			databaseType := ""
```

Comment:
I would add this where it is used (before the loop) as we initialise something too early as many things can fail